packages/server/modules/workspaces/repositories/workspaces.ts

```diff
@@ -161,24 +166,36 @@ export const upsertWorkspaceRoleFactory =
 
     await tables
       .workspacesAcl(db)
-      .insert({ userId, workspaceId, role })
+      .insert({ userId, workspaceId, role, createdAt, updatedAt })
       .onConflict(['userId', 'workspaceId'])
-      .merge(['role'])
+      .merge(['role', 'createdAt', 'updatedAt'])
   }
 
 export const getWorkspaceCollaboratorsFactory =
   ({ db }: { db: Knex }): GetWorkspaceCollaborators =>
-  async ({ workspaceId, filter = {} }) => {
+  async ({ workspaceId, filter = {}, cursor, limit = 25 }) => {
     const query = DbWorkspaceAcl.knex(db)
       .select<Array<UserWithRole & { workspaceRole: WorkspaceRoles }>>([
         ...Users.cols,
         knex.raw(`${DbWorkspaceAcl.col.role} as "workspaceRole"`),
```

Comment:
1. Why do 'withoutTablePrefix' if you do actually add the table prefix, just manually?
2. You can use DbWorkspaceAcl.colAs() to alias columns
```
DbWorkspaceAcl.colAs('role', 'workspaceRole'),
DbWorkspaceAcl.colAs('createdAt', 'workspaceCreatedAt'),
DbWorkspaceAcl.colAs('updatedAt', 'workspaceUpdatedAt')
```

But that's all moot, cause you don't need to avoid aliases and you don't need to alias columns either - because the table aliases are already there. So if you do orderby() or groupby() DbWorkspaceAcl.col.createdAt, its already distinct from any other kind of createdAt because of the col prefix

Reply:
kind of silly, in retrospect. ty for breaking it down

Reply:
I guess it may not be obvious, but DBHelper.col.X does add table prefixes by default, so you don't need to manually ever do that. Its only really for update calls where you want to use .withoutTablePrefix., cause `SET X = Y` instructions can't have table prefixes in them